src/app/new-page/component/content-spacer/content-spacer.component.html

```diff
@@ -1 +1,6 @@
 <!-- Content Spacer Goes Here -->
+<div
+  class="spacer"
+  *ngIf="mode !== 'AUTO'"
```

Comment:
can we change this to be `mode === 'FIXED'`?

I'd rather do that in case we ever come up with a different spacer mode in the future